docs/apache-airflow/core-concepts/objectstorage.rst

```diff
@@ -38,6 +44,21 @@ scheme.
     it depends on ``aiobotocore``, which is not installed by default as it can create dependency
     challenges with ``botocore``.
 
+Cloud Object Stores are not real file systems
```

Comment:
Big +10 on that. People do not understand that and have weird expectations about object storages.